pkg/api/types.go

```diff
@@ -2428,6 +2428,13 @@ type PodSecurityContext struct {
 	// for that container.
 	// +optional
 	RunAsUser *int64
+	// The GID to run the entrypoint of the container process.
+	// Defaults to group specified in image metadata if unspecified.
```

Comment:
> Defaults to group specified in image metadata if unspecified.

That's only a half true as per proposal. It defaults to image metadata only when both fields (RunAsUser and RunAsGroup) unspecified. Please, update fields' descriptions (or the original proposal).